release/README.md

```diff
@@ -54,9 +53,9 @@ script can take about an hour or so to run (depending on how long the tests
 take). Every time you re-run the script, it pulls from the branch and runs
 everything from scratch.
 
-At the end it will print two things, the list of authors, and the md5 sums.
```

Comment:
I would leave "At" here and change the "things," to "things:" and remove the comma after "authors".